src/window/window_wrapper.rs

```diff
@@ -574,7 +603,8 @@ impl WinitWindowWrapper {
             self.renderer.grid_renderer.grid_scale
         );
 
-        window.set_blur(window_blurred && transparency < 1.0);
+        self.apply_blur(window_blurred && transparency < 1.0, None);
+
```

Comment:
This breaks the blur support on Wayland, and possible other platforms that implement it in the future.

Reply:
Nope it doesn't. the `apply_blur` has its own definition for non mac_os use cases which make it working just as before.